python-sdk/tutorials/deploy-with-prebuilt-docker-images/pytorch-prebuilt-docker-image.ipynb

```diff
@@ -0,0 +1,938 @@
+        },
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 0.068634,
+          "end_time": "2021-05-18T20:36:45.559675",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:45.491041",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Initialize workspace\n",
+        "Initialize a [Workspace](https://docs.microsoft.com/azure/machine-learning/service/concept-azure-machine-learning-architecture#workspace) object from the existing workspace you created in the Prerequisites step. `Workspace.from_config()` creates a workspace object from the details stored in `config.json`."
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 0.044633,
+          "end_time": "2021-05-18T20:36:45.655492",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:45.610859",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from azureml.core.workspace import Workspace\n",
+        "\n",
+        "ws = Workspace.from_config()\n",
+        "print(\n",
+        "    \"Workspace name: \" + ws.name,\n",
+        "    \"Azure region: \" + ws.location,\n",
+        "    \"Subscription id: \" + ws.subscription_id,\n",
+        "    \"Resource group: \" + ws.resource_group,\n",
+        "    sep=\"\\n\",\n",
+        ")"
+      ],
+      "outputs": [],
+      "execution_count": null,
+      "metadata": {
+        "collapsed": true,
+        "gather": {
+          "logged": 1620920636972
+        },
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 1.451004,
+          "end_time": "2021-05-18T20:36:47.168399",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:45.717395",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Register model\n",
+        "\n",
+        "Register a file or folder as a model by calling [Model.register()](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.model.model?view=azure-ml-py#register-workspace--model-path--model-name--tags-none--properties-none--description-none--datasets-none--model-framework-none--model-framework-version-none--child-paths-none--sample-input-dataset-none--sample-output-dataset-none--resource-configuration-none-)\n"
```

Comment:
our SDK docs are often incorrect or misleading, and our links often break - would recommend removing

Reply:
done